networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc

```diff
@@ -15,6 +15,11 @@ The external network infrastructure must be configured correctly to route traffi
 
 * Your cluster is configured with ExternalIPs enabled. For more information, read xref:../../networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc#configuring-externalip[Configuring ExternalIPs for services].
 
+[NOTE]
+====
+Do not use the same ExternalIP for EgressIP.
```

Comment:
```suggestion
Do not use the same ExternalIP for the egress IP.
```

In my suggestion, I used "egress IP," but use either "egress IP" or "`EgressIP object`", depending on your intended meaning.

Reply:
Addressed